types/shapediver/appbuildertypecheck.ts

```diff
@@ -481,13 +481,23 @@ const IAppBuilderTabSchema = z
 const IAppBuilderAnchor3dContainerPropertiesSchema = z.object({
 	id: z.string(),
 	location: z.array(z.number()),
-	allowPointerEvents: z.preprocess((val) => {
-		if (val === "true") return true;
-		if (val === "false") return false;
-		return val;
-	}, z.boolean().optional()),
+	allowPointerEvents: z.boolean().optional(),
 	justification: z.nativeEnum(TAG3D_JUSTIFICATION).optional(),
 	previewIcon: z.nativeEnum(IconTypeEnum).optional(),
+	width: z.union([z.string(), z.number()]).optional(),
+	height: z.union([z.string(), z.number()]).optional(),
+});
+
+// Zod type definition for IAppBuilderAnchor2dContainerProperties
+const IAppBuilderAnchor2dContainerPropertiesSchema = z.object({
+	id: z.string(),
+	location: z.array(z.union([z.string(), z.number()])),
```

Comment:
@MajorMeerkatThe3rd is this always a 2D location, correct? If so, can we restrict the length to two? Analogous for the 3d anchors.